tests/test_frame.py

```diff
@@ -1020,9 +1020,17 @@ def test_read_excel() -> None:
         # https://github.com/pandas-dev/pandas-stubs/pull/33
         df11: pd.DataFrame = pd.read_excel("foo")
         df12: pd.DataFrame = pd.read_excel("foo", sheet_name="sheet")
-        df13: Dict[Union[int, str], pd.DataFrame] = pd.read_excel(
+        df13: pd.DataFrame = pd.read_excel("foo", sheet_name=0)
+        df14: Dict[Union[int, str], pd.DataFrame] = pd.read_excel(
```

Comment:
Can you leave `df13` as it was?  (now your `df14`) and make `df14` what you are now calling `df13` so we see the additional tests?

Then above the (new) `df14` add a reference to the Github issue, e.g., # GH 98